src/EnergyPlus/DataDaylighting.hh

```diff
@@ -190,9 +192,9 @@ namespace Dayltg {
         Array1D<Real64> BacLum;                  // =0.0 ! Background luminance at each reference point (cd/m2)
         Array2D<Real64> SolidAngAtRefPt;         // (Number of Zones, Total Daylighting Reference Points)
         Array2D<Real64> SolidAngAtRefPtWtd;      // (Number of Zones, Total Daylighting Reference Points)
-        Array3D<Real64> IllumFromWinAtRefPt;     // (Number of Zones, 2, Total Daylighting Reference Points)
-        Array3D<Real64> BackLumFromWinAtRefPt;   // (Number of Zones, 2, Total Daylighting Reference Points)
-        Array3D<Real64> SourceLumFromWinAtRefPt; // (Number of Zones, 2, Total Daylighting Reference Points)
+        Array2D<std::array<Real64, (int)DataSurfaces::WinCover::Num>> IllumFromWinAtRefPt;     // (Number of Zones, 2, Total Daylighting Reference Points)
```

Comment:
Convert from `Array3D<X>` to `Array2D<std::array<X, 2>>`. This enables shortcuts and makes address calculations within the outer array faster. Mostly the shortcuts though.